tests/test_efadv.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
```

Comment:
please add this file to the tests/CMakeLists (and keep it sorted alphabetically)

Reply:
Done